### PR TITLE
bugfix: subscribe params order

### PR DESCRIPTION
**Pull request by Gerrard-YNWA**
Correct the params' order in `redoSubscribe`.
According to [util.GetGroupName](https://github.com/nacos-group/nacos-sdk-go/blob/2.0.0/util/common.go#L53), first element should be `groupName` and the second one is `serviceName`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/nacos-group/nacos-sdk-go/v2
 go 1.14
 
 require (
+	bou.ke/monkey v1.0.2 // indirect
```
Comment (sanxun0325) [inline go.mod:6]: Can unit tests be done without introducing monkey

**Review by sanxun0325 — COMMENTED** [review]
Please resolve the conflict

File: go.mod (hunk 1 above)
Comment (Gerrard-YNWA) [reply go.mod:6]: Hi @sanxun0325, I notice that `ConnectionEventListener` struct contains a `*NamingGrpcProxy` field and the `NewConnectionEventListener` constructor requires a `NamingGrpcProxy` pointer as input. Can you give me an example for how to mock a `NamingGrpcProxy` instance?

Comment (Gerrard-YNWA) [reply go.mod:6]: Hi @sanxun0325, I notice that `ConnectionEventListener` struct contains a `*NamingGrpcProxy` field and the `NewConnectionEventListener` constructor requires a `NamingGrpcProxy` pointer as input. Can you give me an example for how to mock a `NamingGrpcProxy` instance?

Comment (sanxun0325) [reply go.mod:6]: @Gerrard-YNWA you can refer to this:https://github.com/nacos-group/nacos-sdk-go/blob/2.0.0/clients/naming_client/naming_grpc/connection_event_listener_test.go

Comment (Gerrard-YNWA) [reply go.mod:6]: Hi @sanxun0325 thanks a lot,  `ConnectionEventListener` use `INamingProxy` interface now, Can I use gomock to write test case?

Comment (Gerrard-YNWA) [reply go.mod]: Hi @sanxun0325, I have updated the test case

**Review by sanxun0325 — APPROVED** [review]
Thanks for your contribution,LGTM

File: go.mod [hunk 2]
```diff
@@ -5,19 +5,13 @@ go 1.14
...
 	github.com/go-errors/errors v1.0.1
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.4.2
-	github.com/jehiah/go-strftime v0.0.0-20171201141054-1d33003b3869 // indirect
-	github.com/jonboulle/clockwork v0.1.0 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
-	github.com/lestrrat/go-envload v0.0.0-20180220120943-6ed08b54a570 // indirect
 	github.com/lestrrat/go-file-rotatelogs v0.0.0-20180223000712-d3151e2a480f
-	github.com/lestrrat/go-strftime v0.0.0-20180220042222-ba3bf9c1d042 // indirect
+	github.com/nacos-group/nacos-sdk-go v1.0.9
```
[the remark this answers is not in the document]
Comment (sanxun0325) [reply go.mod]: done

**Pull request merged**